getting-started/installation/general-installation/installation-coreos.markdown

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Installing Enterprise on CoreOS
```

Comment:
@olehermanse Perhaps this should be re-titled to `Installing Enterprise using disk image` Or actually, maybe we need to drop this page completely. I don't see that we are still publishing a disk image since CoreOS is died (and now it's resurrected under Fedora https://fedoraproject.org/coreos/ ).

Reply:
@nickanderson, it seems GitHub expects this review comment to be resolved for the review to be marked complete.
The link against requested change https://github.com/cfengine/documentation/pull/3355#pullrequestreview-2348420447 which doesn't seem like the issue.

Can this be reconsidered?

Reply:
Hi @me-coder I don't think it's because of this comment, just the other comments and suggestions in general where my review was set for requested changes. 

Anyway, this comment had nothing to do with your PR. I just need to re-review.

Reply:
We have deprecated support for CoreOS as of 3.24.x series so we should likely edit this page to specify only available in 3.21.x LTS and after that will be unsupported.

Reply:
I believe, this whole page can be safely dropped in that case for 3.24.x (should be branch specific, if I understand it correctly):
https://docs.cfengine.com/docs/3.24/getting-started-installation-general-installation-installation-coreos.html

I'll rebase it once the deletion is made in some relevant PR, so it automatically reflects here.